pkg/bpf/bpf.go

```diff
@@ -351,6 +351,37 @@ func getNodePodSubGateway(node *corev1.Node) [16]byte {
 	return podGateway
 }
 
+func GetKmeshConfigMap(bpfMap *ebpf.Map) (*KmeshBpfConfig, error) {
+	if bpfMap == nil {
+		return nil, fmt.Errorf("get KmeshConfigMap failed: bpfMap is nil")
+	}
+
+	key := uint32(0)
+	value := KmeshBpfConfig{}
+	if err := bpfMap.Lookup(&key, &value); err != nil {
+		return nil, err
+	}
+
+	return &value, nil
+}
+
+func UpdateKmeshConfigMap(bpfMap *ebpf.Map, config *KmeshBpfConfig) error {
+	if bpfMap == nil {
+		return fmt.Errorf("update KmeshConfigMap failed: KmeshConfigMap is empty")
```

Comment:
```suggestion
		return fmt.Errorf("update KmeshConfigMap failed: bpfMap is nil")
```

Be consistent with `GetKmeshConfigMap`